src/bin/identities/CommandAuthenticate.ts

```diff
@@ -45,32 +45,21 @@ class CommandAuthenticate extends CommandPolykey {
         });
         const providerMessage = new identitiesPB.Provider();
         providerMessage.setProviderId(providerId);
-        providerMessage.setMessage(identityId);
-        const successMessage = await binUtils.retryAuthentication(
-          async (auth) => {
-            const stream = pkClient.grpcClient.identitiesAuthenticate(
-              providerMessage,
-              auth,
-            );
-            const codeMessage = (await stream.next()).value;
-            process.stdout.write(
-              binUtils.outputFormatter({
-                type: options.format === 'json' ? 'json' : 'list',
-                data: [`Your device code is: ${codeMessage!.getMessage()}`],
-              }),
-            );
-            return (await stream.next()).value;
-          },
-          meta,
-        );
-        process.stdout.write(
-          binUtils.outputFormatter({
-            type: options.format === 'json' ? 'json' : 'list',
-            data: [
-              `Successfully authenticated user: ${successMessage!.getMessage()}`,
-            ],
-          }),
-        );
+        providerMessage.setIdentityId(identityId);
+        await binUtils.retryAuthentication(async (auth) => {
+          const stream = pkClient.grpcClient.identitiesAuthenticate(
+            providerMessage,
+            auth,
+          );
+          const codeMessage = (await stream.next()).value;
+          process.stdout.write(
```

Comment:
Do this outside of the lambda. Take the result from `retryAuthentication` and do this.

Reply:
The `retryAuthentication` should give back the stream. Then use the stream outside of `retryAuthentication`. We only retry upon establishing the stream.

Reply:
Done